app/routes/_public.undercut.tsx

```
@@ -1,339 +1,336 @@
+              />
+            </div>
+
+            <InputWithLabel
+              type="number"
+              labelTitle="Stop alerting after:"
+              inputTag="Hours"
+              value={config.ignore_data_after_hours}
+              onChange={(e) =>
+                setConfig((prev) => ({
+                  ...prev,
+                  ignore_data_after_hours: parseInt(e.target.value, 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)